### PR TITLE
[Testing] api testing is done

### PR DESCRIPTION
**Pull request by hanyulo**
Complete API Testing.

### CODE REVIEW

File: api/actions/search-authors.js [hunk 1]
```diff
@@ -9,10 +9,11 @@ const _ = {
   omit
 }
 
-export function searchAuthors(req, path) { // eslint-disable-line no-unused-vars
-  const { ALGOLIA_APP_ID, ALGOLIA_SEARCH_API_KEY } = config
-  let client = algoliasearch(ALGOLIA_APP_ID, ALGOLIA_SEARCH_API_KEY)
-  let index = client.initIndex(CONTACTS_INDEX)
+const { ALGOLIA_APP_ID, ALGOLIA_SEARCH_API_KEY } = config
+let client = algoliasearch(ALGOLIA_APP_ID, ALGOLIA_SEARCH_API_KEY)
+export let index = client.initIndex(CONTACTS_INDEX)
```
Comment (nickhsine) [inline api/actions/search-authors.js]: You can not export `index`

File: api/actions/search-posts.js [hunk 2]
```diff
@@ -9,10 +9,11 @@ const _ = {
   omit
 }
 
-export function searchPosts(req, path) { // eslint-disable-line no-unused-vars
-  const { ALGOLIA_APP_ID, ALGOLIA_SEARCH_API_KEY } = config
-  let client = algoliasearch(ALGOLIA_APP_ID, ALGOLIA_SEARCH_API_KEY)
-  let index = client.initIndex(POSTS_INDEX)
+const { ALGOLIA_APP_ID, ALGOLIA_SEARCH_API_KEY } = config
+let client = algoliasearch(ALGOLIA_APP_ID, ALGOLIA_SEARCH_API_KEY)
+export let index = client.initIndex(POSTS_INDEX)
```
Comment (nickhsine) [inline api/actions/search-posts.js]: You can not export `index`

Comment (hanyulo) [inline api/actions/search-posts.js]: Already modified it. Thx

File: api/actions/__test__/search-authors.test.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      page: 0,
+      nbPages: 8,
+      hitsPerPage: 24,
+      processingTimeMS: 1,
+      exhaustiveNbHits: true,
+      query: '',
+      params: 'query=&filters=articlesCount%3E0&hitsPerPage=24&page=0'
+    }
+    error = new Error('This testing error')
+    stubedIndex = sinon.stub(index, 'search')
```
Comment (hanyulo) [inline api/actions/__test__/search-authors.test.js]: 👌 👌 👌

File: package.json [hunk 4]
```diff
@@ -215,6 +215,7 @@
     "rimraf": "^2.5.2",
     "sinon": "^1.17.1",
     "sinon-chai": "^2.8.0",
+    "sinon-stub-promise": "^4.0.0",
```
Comment (nickhsine) [inline package.json]: Update yarn.lock as well

**Pull request merged**